code/game/objects/items/crayons.dm

```diff
@@ -497,7 +504,8 @@
 	paint_color = null
 
 	item_state = "spraycan"
-	desc = "A metallic container containing tasty paint."
+	desc = "A metallic container containing tasty paint.\n\
+		Alt-click to toggle the cap."
```

Comment:
i'd kinda prefer if you did this like i did in https://github.com/tgstation/tgstation/pull/17050 for the sake of consistency